tests/integration/test_charm_one_runner.py

```diff
@@ -17,7 +17,7 @@
     start_test_http_server,
     wait_till_num_of_runners,
 )
-from tests.status_name import ACTIVE_STATUS_NAME
+from tests.status_name import ACTIVE_STATUS_NAME, BLOCKED_STATUS_NAME
```

Comment:
Should these just be called `ACTIVE`, `BLOCKED` etc since status name is already in the module name?

Reply:
Fixed.